iso/build.sh

```diff
@@ -31,22 +31,25 @@ docker run $DOCKER_RUN_OPTIONS jpetazzo/nsenter cat /nsenter > $tmpdir/nsenter &
 # do not remove nsenter, as this image is not big, and quite generally used
 
 # Get socat
-docker build -t socat -f Dockerfile.socat .
-docker run $DOCKER_RUN_OPTIONS socat cat socat > $tmpdir/socat
+[ "${BUILD_CONTAINER_IMAGES+1}" ] && docker build -t minishift/b2d-socat -f Dockerfile.socat .
```

Comment:
Why the  +1 in `${BUILD_CONTAINER_IMAGES+1}` .  If anyone wants to build the images locally we can set export BUILD_CONTAINER_IMAGES=1, right?